wallet-connect-test-bench/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "test-bench-for-wallets",
+    "packageManager": "yarn@3.2.0",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "engines": {
+        "node": ">=16.x"
+    },
+    "dependencies": {
+        "@concordium/browser-wallet-api-helpers": "2.4.0",
+        "@concordium/web-sdk": "^3.4.0",
+        "@walletconnect/qrcode-modal": "1.8.0",
+        "@walletconnect/sign-client": "2.1.4",
+        "@walletconnect/types": "^2.1.4",
+        "eslint": "^8.37.0",
+        "moment": "^2.29.4",
+        "react": "^18.1.0",
+        "react-dom": "^18.1.0",
+        "react-switch": "^7.0.0"
+    },
+    "devDependencies": {
+        "@craftamap/esbuild-plugin-html": "^0.4.0",
+        "@types/node": "^18.7.23",
+        "@types/react": "^18.0.9",
+        "@types/react-dom": "^18.0.5",
+        "@typescript-eslint/eslint-plugin": "^5.20.0",
+        "@typescript-eslint/parser": "^5.20.0",
+        "cross-env": "^7.0.3",
+        "esbuild": "^0.14.42",
+        "esbuild-plugin-svgr": "^1.0.1",
+        "eslint": "7",
+        "eslint-config-airbnb": "^19.0.4",
+        "eslint-config-airbnb-typescript": "^17.0.0",
+        "eslint-config-prettier": "^8.5.0",
+        "eslint-plugin-import": "^2.26.0",
+        "eslint-plugin-jsx-a11y": "^6.5.1",
+        "eslint-plugin-prettier": "^4.0.0",
+        "eslint-plugin-react": "^7.29.4",
+        "eslint-plugin-react-hooks": "^4.4.0",
+        "fs": "^0.0.1-security",
+        "live-server": "^1.2.2",
```

Comment:
I change the version as suggested and also added a section to the Read.me file.


When I run into the above problems when using an old `yarn` version and resolve it usually by:

> If you see an error message similar to this when executing `yarn start`, then you've run into the problem:
> ```shell
> env: node\r: No such file or directory
> ```
> 
> Use `npm install` instead of `yarn install` in the above command or use an up-to-date `yarn` version. `npm` (newer `yarn` version will correct the line ending).

Reply:
I will address it in next PR.